### ISSUE TITLE
[Bug Report] Clear and date icons are mixed up

### ISSUE THREAD

**Issue by Kater-auf-Dach** [onesine/react-tailwindcss-datepicker#51]
CloseIcon shows when input has no value. And vice versa.
This line seems to be the reason:
`e.isEmpty ? <CloseIcon className="h-5 w-5" /> : <DateIcon className="h-5 w-5" />`

**Comment by AnalogMemory** [comment]
Seeing the same thing

**Comment by AnalogMemory** [comment]
A workaround for this is to use the `toggleIcon` with your own icons or text

```
toggleIcon={isEmpty => {
  return isEmpty ? "Select Date" : "Clear";
}}
```

**Comment by AnalogMemory** [comment]
Looks like this was fixed in release 1.4.2 🎉 
- https://github.com/onesine/react-tailwindcss-datepicker/pull/52

**Comment by onesine** [comment]
> Looks like this was fixed in release 1.4.2 🎉
> 
> * [[Fix] Condition for icons render in `ToggleButton` . #52](https://github.com/onesine/react-tailwindcss-datepicker/pull/52)

It seems that this is already supported in version  1.4.2 🎉. So we can close this issue.

Thanks to @Kater-auf-Dach for the PR and to @AnalogMemory  for the alternative proposed in this discussion.
